index.js

```diff
@@ -17,28 +17,39 @@ function getConfigValue({ configName, defaultValue, env, input }) {
 
 module.exports = {
     onPreBuild: ({ utils, inputs }) => {
-      const expression = getConfigValue({
-        configName: 'expression',
-        defaultValue: '* * * * *',
-        env: process.env.DEPLOYMENT_HOURS_EXPRESSION,
-        input: inputs.expression
-      });
 
-      const timezone = getConfigValue({
-        configName: 'timezone',
-        defaultValue: 'America/Toronto',
-        env: process.env.DEPLOYMENT_HOURS_TIMEZONE,
-        input: inputs.timezone
+      const productionOnly = getConfigValue({
+        configName: 'productionOnly',
+        defaultValue: false,
+        env: process.env.DEPLOYMENT_HOURS_PRODUCTION_ONLY,
+        input: inputs.productionOnly
       });
 
-      const cr = getCronAllowedRange(expression, timezone, utils);
-      const now = new Date();
-
-      console.log(`Current time: '${now}'. Expression: '${expression}'. Timezone: '${timezone}'.`);
-
-      if(!cr.isDateAllowed(now)) {
-        utils.build.cancelBuild('Deployment not allowed at this time.');
+      if(productionOnly === false || (productionOnly === true && process.env.CONTEXT === 'production')){
```

Comment:
Could we exit early instead? E.g.

```js
if((productionOnly === true && process.env.CONTEXT !== 'production') {
  console.log('Skipping deployment hour check because process.env.CONTEXT is '${process.env.CONTEXT}'.);
  return;
}
```